packages/snaps-execution-environments/src/common/endowments/endowments.test.browser.ts

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable import/no-unassigned-import, @typescript-eslint/restrict-template-expressions */
```

Comment:
I think this is a bug in `webdriverio`. These tests do run, you can verify by looking at the number of total tests. It happens for more tests, not just the ones declared in a loop.

Reply:
I think it's important to have a feedback that these tests are running and passing. It was previously listing all the endowments that are tested for two security checks (hardening and search for globalThis).

Is there maybe another way in which this can be refactored to make it work with `webdriverio` in such way? Scaffolding used with AVA seems not suitable for the `webdriverio`.

Reply:
> Is there maybe another way in which this can be refactored to make it work with webdriverio in such way? Scaffolding used with AVA seems not suitable for the webdriverio.

It's not an issue with the test. Mocha supports this pattern as well. Webdriverio just doesn't seem to list all the tests that are running (even the ones that are declared normally). If you know of a way to fix this, please let me know!

Reply:
Yea, got it now, this is some kind of bug in spec reporter.

But, I found something that can fix this in some way. So, I propose adding the following configuration for the `snaps-execution-environment` only (`/snaps-execution-environments/wdio.config.ts`):

```javascript
{
  ...
  reporters: [
    [
      'spec',
      {
        addConsoleLogs: true,
        showPreface: false,
        realtimeReporting: true,
      },
    ],
  ],
}
```

This will output everything in a real time and make it clear what's being tested. I know it will provide a lot more output, but seems to be somehow more confident when I know that none of my tests are being skipped.
![Screenshot 2023-02-27 at 17 57 24](https://user-images.githubusercontent.com/13301024/221629104-3ee79771-7cff-4c65-97bf-c406ea024560.png)

This is almost the same way how AVA was reporting these tests.